pkgs/development/cuda-modules/packages/cuda_nvcc.nix

```diff
@@ -144,13 +149,19 @@ buildRedist (finalAttrs: {
         EOF
       ''
       # Add the dependency on backendStdenv.cc to the nvcc.profile.
+      # NOTE: NVCC explodes in horrifying fashion if GCC is not on PATH -- it fails even before
+      # reading nvcc.profile!
       + ''
-        nixLog "adding backendStdenv.cc to nvcc.profile"
+        nixLog "setting compiler-bindir to backendStdenv.cc in nvcc.profile"
         cat << EOF >> "''${!outputBin:?}/bin/nvcc.profile"
-
         # Fix a compatible backend compiler
-        PATH += "${backendStdenv.cc}/bin":
+        compiler-bindir = ${backendStdenv.cc}/bin
```

Comment:
Ah damn this is off by 1 again: nvcc here is `hostTarget`, and `cc` - `buildHost`

Reply:
Nooo that's the thing about `stdenv`s, they're always offset by -1:

```console
$ nix eval -f "<nixpkgs>" --argstr crossSystem aarch64-linux stdenv.cc.name
"aarch64-unknown-linux-gnu-gcc-wrapper-14.3.0"
```

But maybe we should merge regardless and fix whenever